fmt/src/formatter.rs

```diff
@@ -1977,6 +1959,108 @@ impl<'a, W: Write> Visitor for Formatter<'a, W> {
 
         Ok(())
     }
+
+    fn visit_var_definition_stmt(
+        &mut self,
+        _loc: Loc,
+        declaration: &mut VariableDeclaration,
+        expr: &mut Option<Expression>,
+        semicolon: bool,
+    ) -> Result<()> {
+        declaration.visit(self)?;
```

Comment:
I was thinking about this more as I came across it in the expression stuff. I think we may have to inline the `visit_var_declaration` here, because the variable identifier needs to be chunked with the `equals` sign because we don't want them to be separated onto different lines. also this is the only place where we use `visit_var_declaration` anyways... In addition, we have to check if the var declaration ends up on multiple lines, because in that case we have to indent the right side of the expression in case it also ends up on multiple lines. If you want however, I can just handle those cases in the fmt expression branch

Reply:
yeah, just leave it as is, and I'll implement it in the fmt expressions stuff!